docs/06-documentation-and-release-notes/files/README-for-Demos.md

```diff
@@ -0,0 +1,130 @@
+# README.md Template for Demos
+
+# [Connector_Name] Anypoint&trade; Studio Connector Demo - [Demo_Name]
```

Comment:
Why the "Anypoint" and "Studio"? Surely demo users know they are dealing with a project that is supposed to be imported into Anypoint Studio. :-)

Jokes aside, I think it's redundant and is an overuse of branding. I would just leave it as "[Connector_Name] Connector Demo - [Demo_Name]".